pkg/deployment/resources/pod_creator_arangod.go

```diff
@@ -66,7 +66,8 @@ type MemberArangoDPod struct {
 	resources        *Resources
 	imageInfo        api.ImageInfo
 	autoUpgrade      bool
-	id               string
+	args             []string
+	volumes          pod.Volumes
```

Comment:
Also here, I would vote to generate it on demand

Reply:
changed